modules/apps/segments/segments-web/src/main/resources/META-INF/resources/types/Criteria.d.ts

```diff
@@ -12,17 +12,24 @@
  * details.
  */
 
-import {DateValue} from "./Date";
+import type {Conjunction, PropertyType} from "../js/utils/constants";
+
+export interface Property {
+	label: string;
+	name: string;
+	type: PropertyType;
+}
 
 export interface CriteriaItem {
```

Comment:
Hi @p2kmgcl , why not leaving the assetId, day and operatorNot keys as optional? In an event criteria those keys could be present.

Reply:
Because I didn't know that :sweat_smile:, updating!!!